### PR TITLE
Doc: Draft overlay icons

### PR DESCRIPTION
**Pull request by dragotin**
@owncloud/desktop-developers please check if that is common understanding.

@MTRichards does that reflect your understanding of the overlay icons 

@carlaschroder draft input for the docs, please do whatever you want with it ;-)

### CODE REVIEW

**Comment by carlaschroder** [comment]
Hurrah, I love when someone else writes stuff! Editing is so much easier. Thanks @dragotin!

**Comment by carlaschroder** [comment]
Hmm, Github doesn't show a diff of my changes. I hope that doesn't cause any confusion.

File: doc/navigating.rst [hunk 1]
```diff
@@ -133,6 +133,44 @@ appear as a new tab in the settings dialog, where you can adjust its settings
...
+
+If there is a sync error, or the file is blacklisted, it displays an 
+eye-catching red X.
+
+If the file is waiting to be synced, or is currently syncing, the overlay 
+icon displays a blue cycling icon.
+
+When the client is offline, the icons remain in the state of the 
+last successful sync, as long as they are not touched locally. Once a file
+is edited locally, the icon will change to the 'waiting to sync' icon.
```
Comment (jturcotte) [inline doc/navigating.rst]: The idea of showing a waiting to sync icon while offline is incompatible with the waiting to sync status bubbling up to parent directories between syncs, unless you'd like to do a local discovery on startup only to show what is waiting for sync vs the database state (since we can't afford a recursive scan of the whole tree).

Just hiding all overlay icons while offline/paused would allow things to be simpler and bug-free. The whole point of refactoring this code was to improve the reliability on OS X since the file status was often getting stuck in a sync or OK state.

Comment (dragotin) [inline doc/navigating.rst]: @jturcotte understood.  Lets do what you suggest and display no icon when being disconnected.

For the future, I think we should display the disconnected icon at least on the root sync folder. I will create a separate issue.

Comment (MTRichards) [inline doc/navigating.rst]: @dragotin read through it and it makes sense to me.

File: doc/navigating.rst [hunk 2]
```diff
@@ -133,6 +133,44 @@ appear as a new tab in the settings dialog, where you can adjust its settings
...
+
+The overlay icon of a synced directory indicates the status of the files in the 
+directory. If there are any sync errors, the directory is marked with a warning 
+icon.
+
+If the files inside the directory include ignored files that are marked with 
+warning icons, the parent directory remains in green mode.
```
Comment (ogoffart) [inline doc/navigating.rst]: There will be a warning sign in case of error, even if there is ignored files.  The current wording seems to indicate that if there are ignored files, then the parent directory will always be green (even if there are other errors.)

Maybe reword it like : "include ignored files that are marked with warning icons does not change the status of the parent directies"

File: doc/navigating.rst [hunk 3]
```diff
@@ -133,6 +133,44 @@ appear as a new tab in the settings dialog, where you can adjust its settings
...
+
+Normally-synced directories will not show an error sign, with the exception of 
+that root sync directory. That will turn red if there is a serious error 
+anywhere inside the root directory.
```
Comment (ogoffart) [inline doc/navigating.rst]: I don't understand this part.   Any directory turns red if there is a "serious" error anywhere into them.  how is the root different?

**Comment by dragotin** [comment]
Applied the feedback from the review here. @carlaschroder feel free to merge please.

File: doc/navigating.rst [hunk 4]
```diff
@@ -133,6 +133,39 @@ appear as a new tab in the settings dialog, where you can adjust its settings
 at 
 any time. Use **Account** > **Remove** to delete accounts. 
 
+File Manager Overlay Icons
+--------------------------
+
+The ownCloud sync client provides overlay icons, in addition to the normal file 
+type icons, for your system file manager (Explorer on Windows, Finder on Mac and
```
Comment (guruz) [inline doc/navigating.rst]: Dolphin?

File: doc/navigating.rst [hunk 5]
```diff
@@ -133,6 +133,39 @@ appear as a new tab in the settings dialog, where you can adjust its settings
...
+file is in sync with the server version, it displays a green checkmark.
+
+If the file is ignored from syncing, for example because it is on your 
+exclude list, or because it is a symbolic link, it displays a warning icon.
+
+If there is a sync error, or the file is blacklisted, it displays an 
+eye-catching red X.
+
+If the file is waiting to be synced, or is currently syncing, the overlay 
+icon displays a blue cycling icon.
```
Comment (guruz) [inline doc/navigating.rst]: I'm not a native speaker, but I'd have said blue icon with white arrows?

Comment (MTRichards) [inline doc/navigating.rst]: >icon displays a blue cycling icon

You could say a blue circular icon, that may be best so it is parallel with previous icon descriptions.

Comment (dragotin) [inline doc/navigating.rst]: almost... ;-)

**Comment by dragotin** [comment]
Thanks, but I'll leave the language fine print to @carlaschroder - I shouldn't even try...

**Comment by carlaschroder** [comment]
Merging. I am wondering if more file managers are supported? @guruz said Dolphin. I'll revise the file manager list later.

**Comment by dragotin** [comment]
Yes, dolphin is also supported, but only in very new versions of KDE Frameworks 5 (>5.16). Only openSUSE Tumbleweed ships that in the range of distros we support, so I wonder if we should already mention it or wait a bit.

**Comment by carlaschroder** [comment]
Meh, wait until it propagates more widely.
